### ISSUE TITLE
scroll sensitivity

### ISSUE THREAD

**Issue by john012343210** [imbushuo/mac-precision-touchpad#503]
how to adjust the scroll sensitivity?

**Comment by waleedrizwan** [comment]
I am also trying to figure this out

**Comment by waleedrizwan** [comment]
Hey I figured out the solution on windows 11. Go to bluetooth and devices and find touchpad settings, and then adjust the cursor speed from there.

**Comment by versatile712** [comment]
> 

Hi, I tried, it only affects the cursor speed but not the scroll speed in my case. Does it work on your Mac that the scroll speed also changes? E.g., going down faster when browsing webpages

**Comment by waleedrizwan** [comment]
I can't find a way to change the scroll speed. Maybe if you try to add a custom three finger gesture you can create a faster scroll

**Comment by kleqing** [comment]
idk if it work or not but you can change it via 'mouse properties'

![image](https://user-images.githubusercontent.com/78801337/224259752-31bcf59d-6f5a-43ce-bcf0-52787fa879ee.png)

Just go to settings->bluetooth & device->more touchpad settings->wheel

**Comment by smaddy** [comment]
yeah cant find a way to adjust the scroll speed. useless without it.
